web-admin/src/routes/[organization]/-/settings/billing/payment/+page.ts

```
@@ -0,0 +1,18 @@
+import { fetchPaymentsPortalURL } from "@rilldata/web-admin/features/billing/plans/selectors";
+import { error, redirect } from "@sveltejs/kit";
+import type { PageLoad } from "./$types";
+
+export const load: PageLoad = async ({ params: { organization }, url }) => {
+  let redirectUrl = `/${organization}/-/settings/billing`;
+  try {
+    redirectUrl = await fetchPaymentsPortalURL(
+      organization,
+      `${url.protocol}//${url.host}/${organization}`,
+    );
+  } catch (e) {
+    if (e.response?.status !== 403) {
```

We should throw the error when it's a 403 too